docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -93,6 +73,20 @@ If you want to continue testing YugabyteDB with more resource-intensive scenario
 
 Currently self-service migration is not supported. Contact [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431) for help with migration.
 
+## YugabyteDB
+
+### What version of YugabyteDB does Yugabyte Cloud run on
+
+New standard clusters are provisioned with the YugabyteDB [stable release](../../releases/whats-new/stable-release/).
+
+Free clusters are provisioned with the [latest release](../../releases/whats-new/latest-release/).
+
+To view the database version running on a particular cluster, navigate to the **Clusters** page; the database version is displayed next to the cluster name.
```

Comment:
For my info: is this true on free clusters, or only paid?

Reply:
Any cluster